stylesheet.css

```diff
@@ -0,0 +1,310 @@
+* {
+  /* border: 1px black dotted; */
+  /* overflow: hidden; */
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+  padding: 0;
+  background: rgb(255, 239, 245);
+  height: 100vh
+}
+
+.container {
+  display: grid;
+  grid-template-areas: "nav main sidebar";
+  grid-template-columns: minmax(225px, 300px) minmax(min-content, 2fr) minmax(290px, 1fr);
+}
+
+.nav {
+  grid-area: "nav";
+  font-weight: bold;
+  color: #454545;
+  overflow: none;
+}
+
+.nav, .sidebar {
+  background-color: rgba(255,255,255,0.3);
+  padding: 5px;
+}
+
+.nav ul {
+  padding: 10px;
+  margin: 0;
+}
+
+.nav ul li {
+  padding: 10px;
+  margin:  0.5em 0;
+  list-style-type: none;
+}
+
+.nav ul li > img + span {
+  margin-left: 10px;
+}
+
+.nav ul li:hover, #active {
+  background-color: rgba(255, 255, 255, 0.8);
+  border-radius: 20px;
+}
+
+.createNew {
+  background-color: rgba(255, 255, 255, 0.8);
+  height: min-content;
+  display: grid;
+  grid-template-columns: 50px 1fr;
+  grid-template-rows: 1fr 40px;
+  padding: 20px;
+  gap: 20px;
+}
+
+.textBox {
+  background-color: rgba(255, 255, 255, 1);
+  padding: 10px;
+  border-radius: 10px;
+  box-shadow: 0px 5px 10px rgba(98, 27, 85, 0.1);
+}
+
+.textBox > p {
+  margin: 0;
+}
+
+.createNew .button {
+  font-weight: bold;
+  padding: 10px;
+  border-radius: 20px;
+  border: 1px solid rgb(236, 72, 135); 
+}
+
+.button:hover {
+  background-color: rgb(236, 72, 135);
+  color: rgba(255, 255, 255, 1);
+}
+
+.createNew .OptionsAndSave {
+  grid-column: 2;
+  display: grid;
+  grid-template-columns: 1fr min-content;
+}
+
+.tweet .postOptions {
+  font-size: small;
+  margin-top: 20px;
+  display: grid;
+  gap: 10px;
+  grid-template-columns: repeat(auto-fill, minmax(45px, 1fr));
+}
+
+.tweet .postOptions img {
+  height: 20px;
+}
+
+.OptionsAndSave .postType {
+  display: grid;
+  grid-template-columns: repeat(6, min-content);
+  grid-gap: 3px;
+  align-items: start;
+  justify-items: center;
+}
+
+.postType img {
+  height: 20px;
+  border-radius: 100%;
+  padding: 10%;
+  margin: 0 5px;
+}
+
+.postType img:hover {
+  background-color: rgb(255, 206, 226);
+}
+
+.tweet {
+  background-color: rgba(255, 255, 255, 0.8);
+  border-radius: 10px;
+  padding: 20px;
+  margin: 20px;
+  display: grid;
+  grid-gap: 20px;
+  grid-template-columns: 50px 1fr;
+  box-shadow: 5px 5px 10px rgba(98, 27, 85, 0.1);
+}
+
+.tweet:hover {
+  cursor: pointer;
+
+  background-color: rgba(255, 255, 255, 1);
+}
+
+.profileName + .handle, .handle + .timestamp {
+  margin-left: 10px;
+}
+
+.tweet .timestamp {
+  font-style: italic;
+  color: rgba(98, 27, 85, 0.5);
+  font-size: smaller;
+}
+
+.tweet .handle {
+  color:rgba(236, 72, 135, 0.5);
+}
+
+.tweet .text {
+  margin-top: 1em;
+  line-height: 150%;
+}
+
+
+.tag, .mention, .profileName, .news p, .followSuggestions p {
```

Comment:
Good use of multiple class names to keep your code DRY. I am terrible at this.

Reply:
It's a fine line to walk because sometimes I'm like "well I want this highlight color in all of these elements" so I could group it but then an alternating background color or some other styling DOESN'T apply and it really fragments the styling for one class – so I'm not really consistent there tbh